vscode/src/webviewPanel.ts

```diff
@@ -44,229 +44,233 @@ export function registerWebViewCommands(context: ExtensionContext) {
   ).toString();
 
   context.subscriptions.push(
-    commands.registerCommand(`${qsharpExtensionId}.showRe`, async () => {
-      clearCommandDiagnostics();
-      const associationId = getRandomGuid();
-      sendTelemetryEvent(
-        EventType.TriggerResourceEstimation,
-        { associationId },
-        {},
-      );
-      const program = await getActiveProgram();
-      if (!program.success) {
-        throw new Error(program.errorMsg);
-      }
+    commands.registerCommand(
+      `${qsharpExtensionId}.showRe`,
+      async (expr?: string) => {
```

Comment:
Is this mostly formatting changes because you need to pass expr?

Reply:
yeah, the only real change is receiving `expr` and then passing it down to the next layer (on line 216 in the new file). It's unfortunate that the formatting changed so drastically, because it makes the diff look like lot's of things changed when they really didn't.